packages/web/src/dev/ValidatorsListApp.tsx

```diff
@@ -0,0 +1,78 @@
+import ApolloClient from 'apollo-boost'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import fetch from 'cross-fetch'
+import gql from 'graphql-tag'
+import getConfig from 'next/config'
+import * as React from 'react'
+import { ApolloProvider, Query } from 'react-apollo'
+import ShowApolloError from 'src/dev/ShowApolloError'
+import ValidatorsList from 'src/dev/ValidatorsList'
+import { I18nProps, withNamespaces } from 'src/i18n'
+
+function createApolloClient() {
+  return new ApolloClient({
+    uri: getConfig().publicRuntimeConfig.BLOCKSCOUT.uri,
+    cache: new InMemoryCache(),
+    // TODO: Remove this workaround when the backend service fixes not needed errors
```

Comment:
just checking if this TODO still applies

Reply:
I think so, the API is having some changes right now, and sometimes is needed. From my point of view, we should keep it there until we stabilize the API.